packages/fast-components-class-name-contracts-base/src/horizontal-overflow/index.ts

```diff
@@ -2,8 +2,23 @@
  * The class name contract for the horizontal overflow component
  */
 export interface IHorizontalOverflowClassNameContract {
+    /**
+     * The root horizontal overflow component - contains overflow items and previous/next controls
```

Comment:
Why mention what this containers here and not other places? I'd suggest removing the second sentence.